server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/BounceTest.java

```diff
@@ -77,10 +77,9 @@ public void bounceShouldReturnAMailToTheSenderWithoutAttributes() throws Excepti
 
         bounce.service(mail);
 
-        FakeMailContext.SentMail expected = new FakeMailContext.SentMail(null,
-            Lists.newArrayList(senderMailAddress),
-            null,
-            ImmutableMap.<String, Serializable>of());
+        FakeMailContext.SentMail expected = new FakeMailContext.SentMail.Builder()
+            .recipients(Lists.newArrayList(senderMailAddress))
```

Comment:
a builder should make argument passing easy, you should not have to create a new list for that